projects/microchip/curiosity_pic32mzef/mplab/aws_tests/.gitignore

```diff
@@ -0,0 +1,4 @@
+Debug
+dist
+build
```

Comment:
Nit: I don't believe it's necessary, but `/` after directories to be consistent with other `.gitignore`s